modules/features/discover/src/test/kotlin/au/com/shiftyjelly/pocketcasts/discover/worker/CuratedPodcastsCrawlerTest.kt

```diff
@@ -30,24 +31,15 @@ class CuratedPodcastsCrawlerTest {
             .addConverterFactory(MoshiConverterFactory.create(moshi))
             .build()
             .create<ListWebService>()
-        crawler = CuratedPodcastsCrawler(service, staticHostUrl = server.url("/static").toString())
-    }
-
-    @Test
-    fun `use specified crawling platform`() = runTest {
-        server.enqueue(MockResponse())
-
-        crawler.crawl("some-platform")
-        val request = server.takeRequest()
-
-        assertEquals("/discover/some-platform/content_v2.json", request.path)
```

Comment:
I don't believe this needs to support other platforms but please let me know if that's wrong.